src/memray/commands/attach.py

```diff
@@ -282,6 +350,22 @@ def prepare_parser(self, parser: argparse.ArgumentParser) -> None:
             choices=["auto", "gdb", "lldb"],
         )
 
+        parser.add_argument(
+            "--mode",
+            help="Activate or deactivate tracking.",
+            type=str,
+            default="activate",
```

Comment:
Let's instead go with a boolean option named `--stop-tracking`, so that the user runs:
```shell
memray attach --stop-tracking $pid
```